src/Lazy.jl

```diff
@@ -147,7 +147,7 @@ next(::List, xs::List) = first(xs), tail(xs)
 Base.show(io::IO, xs::List) =
   foreach(x->print(io,x), ["("] * interpose(xs, " ") * [")"])
 
-function Base.writemime(io::IO, ::MIME"text/plain", xs::List)
+@compat function show(io::IO, ::MIME"text/plain", xs::List)
```

Comment:
The package fails to even load in 0.3, so it should not claim to support it in REQUIRE if support can not be brought back there.